test/images/sample-device-plugin/VERSION

```diff
@@ -1 +1 @@
-1.6
```

Comment:
It's already updated to 1.7, not sure why are you updating or GitHub is allowing us to modify? 🤔

Reply:
PR: https://github.com/kubernetes/kubernetes/pull/123129

Am I missing something here?

Reply:
@mkumatag Thank you for the reveiw!
Sorry, it was a leftover commit from the initial revision of this PR. Should be fixed now.